### PR TITLE
RTL Banners (bug 1148603)

### PR DESCRIPTION
**Pull request by mstriemer**
mkt-banner-animate mov

### CODE REVIEW

**Comment by mstriemer** [comment]
The animation was supposed to happen in the nav update but I forgot to do it. So here it is.

**Comment by mstriemer** [comment]
Related https://github.com/mozilla/marketplace-elements/pull/13

File: src/media/js/main.js [hunk 1]
```diff
@@ -144,7 +144,9 @@ require(
 
     window.addEventListener(
         'resize',
-        _.debounce(function() {z.doc.trigger('saferesize');}, 200),
+        _.debounce(function() {
+            document.dispatchEvent(new CustomEvent('saferesize'));
```
Comment (ngokevin) [inline src/media/js/main.js]: CustomEvent constructor might not be supported in all browsers. I'd include element_utils.MktEvent to make sure it's "polyfill"ed

**Comment by ngokevin** [comment]
r+wc

File: src/media/js/main.js (hunk 1 above)
Comment (mstriemer) [inline src/media/js/main.js]: It's polyfilled in mcm so it should be fine, https://github.com/mozilla/marketplace-core-modules/blob/4d0cbe586999f84d5d83f1ab37895095b79c2b3e/core/polyfill.js#L21-L27.

Comment (mstriemer) [inline src/media/js/main.js]: Which we only seem to require in `settings_local`. That should probably be in init also.

Comment (mstriemer) [inline src/media/js/main.js]: Actuallly `core/init` depends on the polyfill, so we're good.

**Pull request merged**